### DISCUSSION TITLE
Improve a11y/UI for datetime fields on mobile

### DISCUSSION THREAD

**Discussion by thupsi** [craftcms/cms#9903] [Ideas] [answered]
Recently I've been building an app that requires frequent updates via mobile. I have noticed that the date and time fields are not very easy to use on a small touch screen: The touch targets are pretty small and the page moves when the keyboard slides up.

I realize that especially date fields are notoriously difficult to get right. 

I would maybe suggest that these fields switch to the native UI when on mobile? As an example [flatpicker.js](https://flatpickr.js.org/) disables by default the non-native picker on mobile.

![image](https://user-images.githubusercontent.com/19857599/135428604-cd502653-bc55-49d7-b764-fff916ca00d5.png)

**Comment by brandonkelly** [comment, marked as answer]
Good call. Just made that change for the next release (f759541e4036778720baf025a8233ff2ec7f07f3).

**Reply by brandonkelly** [reply]
Craft 3.7.15 is out now with that change.
